samples/examples/FSharpExamples/NaiveBayes.fs

```diff
@@ -143,92 +169,144 @@ type NaiveBayes() =
         let seqCounts = 
             trainSet
             |> Seq.choose chooseLine
-            |> Seq.fold (addWords numClasses) null
+            |> Seq.fold (addWords numClasses) null 
         printfn "Seq train took: %A" (sw.Stop(); sw.Elapsed)
 
         // ...only difference is that the DSet version needs a second "reducer" function
         // to do sum up intermediate per-partition results.
         // DSet.distributeN will create N partitions per node.
         let name = "20News-TinyTest-" + Guid.NewGuid().ToString("D")
-        sw.Restart()
-        let dsetCounts = 
-            DSet<string>(Name = name, Cluster = cluster)
-            |> DSet.distributeN 8 trainSet
-            |> DSet.choose chooseLine
-            |> DSet.fold (addWords numClasses) addCounts null
-        printfn "DSet train took: %A" (sw.Stop(); sw.Elapsed)
 
-        sw.Restart()
-        let mapReduceCounts = naiveBayesMapReduce name cluster trainSet
-        printfn "MapReduce train took: %A" (sw.Stop(); sw.Elapsed)
+        let foldPrajna, mapReducePrajna = trainAndTime cluster name "Prajna formatter" trainSet
```

Comment:
In general, we avoid using "Prajna" in type/member/value names.  We want to only use it in namespace and constants. So in the future, if a rename is needed, it won't be too many changes.